### PR TITLE
New header design

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

**Comment by aintgoin2goa** [comment]
Basic implementation of the new header.

Still to do:
* Nav bar
* Fallbacks for browsers that can't do flexbox

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /npm-debug.log
 
 /.idea/
+.vscode/
```
Comment (i-like-robots) [inline .gitignore]: =]

File: header/_header-new.scss [hunk 2]
```diff
@@ -1,6 +1,148 @@
...
+		}
+	}
+
+	.o-header__left-tools__link-menu:before {
+		@include oIconsGetIcon('hamburger', oColorsGetPaletteColor($icon_color), $icon_size);
+		display: block;
+
+		@include oGridRespondTo(L) {
+			@include oIconsGetIcon('hamburger', oColorsGetPaletteColor($icon_color), $icon_size_l);
+			display: block;
```
Comment (i-like-robots) [inline header/_header-new.scss]: redundant?

File: header/_header-new.scss [hunk 3]
```diff
@@ -1,6 +1,148 @@
...
+		}
+	}
+
+	.o-header__left-tools__link-search:before {
+		@include oIconsGetIcon('search', oColorsGetPaletteColor($icon_color), $icon_size);
+		display: block;
+
+		@include oGridRespondTo(L) {
+			@include oIconsGetIcon('search', oColorsGetPaletteColor($icon_color), $icon_size_l);
+			display: block;
```
Comment (i-like-robots) [inline header/_header-new.scss]: same?

File: header/_header-new.scss [hunk 4]
```diff
@@ -1,6 +1,148 @@
...
+
+		@include oGridRespondTo(M) {
+			display: block;
+		}
+	}
+
+	.o-header__right-tools-link {
+		width: 32px;
+		height: 32px;
+		text-indent: -9999px;
```
Comment (i-like-robots) [inline header/_header-new.scss]: Add an overflow too

File: header/partials/nav-new.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{{#if @root.flags.globalNavigation}}
+<nav id="primary-nav" class="o-header__nav-container" role="navigation"data-trackable="primary-nav"  aria-label="Primary navigation">
+	<ol class="o-header__nav">
+		<li class="o-header__nav-item o-header__meganav-container"
+				data-trackable="group--more"
+				data-on-expanded="noscroll-body">
```
Comment (i-like-robots) [inline header/partials/nav-new.html]: `data-on-expanded="noscroll-body"`

Does this do anything?

File: header/template-new.html [hunk 6]
```diff
@@ -8,32 +8,29 @@
...
+
+			<nav class="o-header__right-tools" role="navigation" aria-label="Account and personalisation links">
+				<ul class="o-header__tools-items">
+					{{#if navigationModel.myFT}}
+					<li class="o-header__right-tools-item" data-tour-stage="myFt">
+						<a class="o-header__right-tools-link" href="/myft" data-trackable="my-ft"><span class="n-util-visually-hidden">myFT</span></a>
+					</li>
+					{{/if}}
+				</ul>
+			</nav>
```
Comment (i-like-robots) [inline header/template-new.html]: I'd argue neither a list nor nav - I reckon marking it up as such would clutter up the document outline

File: header/_header-new.scss (hunk 2 above)
Comment (aintgoin2goa) [inline header/_header-new.scss]: I don't think so - which bit are you referring to?

File: header/_header-new.scss (hunk 3 above)
Comment (aintgoin2goa) [inline header/_header-new.scss]: Again - which bit?  The styles themselves are needed, but I have done it in a stupid way or something?

File: header/partials/nav-new.html (hunk 5 above)
Comment (aintgoin2goa) [inline header/partials/nav-new.html]: probably - this is just copied and pasted from old nav.  Will all be removed at a later date

File: header/_header-new.scss (hunk 3 above)
Comment (i-like-robots) [inline header/_header-new.scss]: Will this not inherit the display rule from above? If not maybe add a comment about why it's needed here again.

**Pull request merged**